packages/wallets/src/components/WalletTourGuide/WalletTourGuideSettings.tsx

```diff
@@ -37,8 +37,9 @@ export const TooltipComponent = ({
         <div {...tooltipProps} className='wallets-tour-guide__container'>
             <div className='wallets-tour-guide__header'>
                 {step?.title as React.ReactNode}
-                <CloseIcon
+                <LegacyClose2pxIcon
                     className='wallets-tour-guide__close-icon'
+                    iconSize='xs'
                     onClick={skipProps.onClick as unknown as React.MouseEventHandler<SVGElement>}
```

Comment:
Not related to your changes but this type looks abit sus 🤨

Reply:
hey @aizad-deriv 
that's how i can select 16x16 size in quill-icons.
maybe i missed smth about how we can do that in other way 🤔
![Screenshot 2024-04-26 at 09 21 28](https://github.com/binary-com/deriv-app/assets/103181646/5f98ebde-ad99-4b7c-a6d4-89f14006e777)

Reply:
sorry I mean this line ` onClick={skipProps.onClick as unknown as React.MouseEventHandler<SVGElement>}` 🙏🏼

Reply:
got it! 
yeah @aizad-deriv i have checked this type, unfortunately we can't fix it because it's going from the react-joyride library type :(